lectures/matrices/README.md

```diff
@@ -0,0 +1,206 @@
+<h2 align=center>Lecture 03</h2>
+
+<h1 align=center>You're getting old, Matrix</h1>
+
+<h3 align=center>10 Prairial, Year CCXXXI</h3>
+
+***Song of the day***: _[**Cupid (Twin Version)**](https://youtu.be/6uvUTu716rU) by FIFTY FIFTY (피프티피프티) (2023)._
+
+### Sections
+
+1. [**The XYZs of Animation**](#part-1-the-xyzs-of-animation)
+2. [**Matrix Arithmetic**](#part-2-matrix-arithmetic)
+    1. [**Scalar Multiplication**](#scalar-multiplication)
+    2. [**Matrix Multiplication**](#matrix-multiplication)
+3. [**Modeling Operations**](#part-3-modeling-operations)
+    1. [**Scaling**](#scaling)
+    2. [**Rotation**](#rotation)
+    3. [**Translation**](#translation)
+4. [**Matrices in OpenGL**](#part-4-matrices-in-opengl)
+
+### Part 1: _The XYZs of Animation_
```

Comment:
Change this title to `The XYZs of Movement`. Technically, transformations aren't animation. I've been meaning to make this change for semesters and I never do lol. Make sure to also reflect the change in line 11.